src/test/javascript/spec/component/modeling-assessment-editor/modeling-assessment-editor.component.spec.ts

```diff
@@ -184,6 +203,38 @@ describe('ModelingAssessmentEditorComponent', () => {
             expect(modelingSubmissionSpy).toHaveBeenCalledOnce();
             modelingSubmissionSpy.mockRestore();
         }));
+        it('call ngOnInit with submissionId set to new', fakeAsync(() => {
+            paramMapSubject.next(
+                convertToParamMap({
+                    submissionId: 'new',
+                    // Include other necessary params here
+                    courseId: '1',
+                    exerciseId: '1',
+                }),
```

Comment:
I think we could omit this comment as it's clear from the usage

```suggestion
                convertToParamMap({
                    submissionId: 'new',
                    courseId: '1',
                    exerciseId: '1',
                }),
```